docs/security/homePerspectives.md

```diff
@@ -0,0 +1,35 @@
+# Home Perspectives
+
+Every time a user logs in to an Uberfire application the system need to resolve what is the default perspective to show up. Next are the steps applied by this resolution procedure:
+
+1.- The authorization policy is asked about the home perspective setting assigned to the user. For example:
+
+```
+role.admin.home=AdminDashboard
+role.manager.home=ManagerDashboard
+```
+For users with _admin_ role the perspective with _id=AdminDashboard_ is taken as the default.
+
+If no home perspective is defined or access is not granted then the next step is executed.
```

Comment:
I don't understand what you mean by "next step" and "executed" in this sentence. The roles are specified declaratively - what is being executed? :-)